spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.feature 'User show page' do
+  let!(:user1) { User.create(name: 'Ajrass', bio: 'A student from Morocco', photo: 'https://avatars.githubusercontent.com/u/130588108?v=4') }
+  let!(:post1) { Post.create(author: user1, title: 'Post 01') }
+  let!(:post2) { Post.create(author: user1, title: 'Post 02') }
+  let!(:post3) { Post.create(author: user1, title: 'Post 03') }
+  let!(:post4) { Post.create(author: user1, title: 'Post 04') }
+
+  scenario 'Displays user Profile picture, username, bio and number of posts' do
+    visit user_path(user1)
+    expect(page).to have_content(user1.name)
+    expect(page).to have_content(user1.bio)
+    expect(page).to have_content("Number of posts: #{user1.posts_counter}")
+  end
```

Comment:
- [ ] Hi @Mike47ip, you listed out four scenarios to be tested in this block but ended up testing only three of them. Please can you kindly add a test that asserts you can see the user's profile picture?